inject/inject.vala

```diff
@@ -439,6 +474,15 @@ namespace Frida.Inject {
 			}
 		}
 
+		public async void call_send (string message) {
+			var msg = new Json.Node.alloc ().init_string (message);
+
+			try {
+				yield rpc_client.call ("recv", new Json.Node[] { msg }, io_cancellable);
+			} catch (GLib.Error e) {
+			}
+		}
```

Comment:
I'd put this after `call_init()` as it's called after it. Or at least we should probably guarantee that it is.